.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,41 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine pytest
+        pip install -r requirements.txt
+        sudo apt-install pylint3
+    - name: Run linter
+      run: pylint3 wheatley/**/*.py
+    - name: Run tests
+      run: python -m pytest
+    - name: Make version file
+      run: |
+        echo ${{ github.ref }} | sed 's:.*/v::' > wheatley/version.txt
+        cat wheatley/version.txt
```

Comment:
Nice touch :+1:

Reply:
Thanks 😊. I have to say that Google was involved with the `sed` command, but if I'd have known that `sed` uses `:` instead of `/` for regexes I'd have figured it out eventually XD.